lib/redbird/plug/session/redis.ex

```diff
@@ -25,15 +25,29 @@ defmodule Plug.Session.REDIS do
     put(conn, generate_random_key(), data, init_options)
   end
   def put(_conn, redis_key, data, init_options) do
-    setex(%{key: redis_key, value: data, seconds: session_expiration(init_options)})
-    redis_key
+    key = redbird_key <> redis_key
```

Comment:
What do you think about a small function to wrap this up? It's a simple function but it names it and also removes duplication in the case that we ever changed the key format.

```elixir
setex(%{
       key: add_namespace(key),
      value: data,
      seconds: session_expiration(init_options)
    })
```